components/QuestionPanel.test.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env jest */
+import React from 'react'
+import { shallow } from 'enzyme'
+import { Button } from 'reactstrap'
+import QuestionPanel from './QuestionPanel'
+
+const makeProps = (
+  queueId,
+  userName,
+  userActiveQuestionId,
+  isUserActiveStaff
+) => ({
+  queueId,
+  user: {
+    name: userName,
+  },
+  userActiveQuestionId,
+  isUserActiveStaff,
+})
+
+describe('<QuestionPanel />', () => {
+  test("doesn't render question panel if user is active staff", () => {
+    const props = makeProps(1, 'Genna', 1, true)
+    const wrapper = shallow(<QuestionPanel {...props} />)
+    expect(wrapper.get(0)).toBeNull()
+  })
+
+  test('renders question panel if user is not active staff', () => {
+    const props = makeProps(1, 'Genna', 1, false)
+    const wrapper = shallow(<QuestionPanel {...props} />)
+    expect(wrapper.find(Button))
```

Comment:
Maybe `expect(wrapper.get(0)).not.toBeNull()` for consistency with the other test case?